lib/broccoli/app-config.js

```diff
@@ -1,7 +1,3 @@
-/* jshint ignore:start */
```

Comment:
I'm surprised that this gets through linting. Implies that maybe ESLint isn't running against 100% of Ember CLI. @Turbo87 Can you chime in here as to why this gets through linting?